Testscripts/Windows/AddHardDisk.ps1

```diff
@@ -357,6 +352,16 @@ function Main {
             Write-LogErr "test parameter '$p' is being ignored because it appears to be malformed"
             continue
         }
+
+        if ( "vhdFormat" -eq $temp[0] ) {
+            $vhdFormat = $temp[1]
+            $vmGen = Get-VMGeneration $VMName $HvServer
+            if ($vmGen -ne 1 -and $vhdFormat -eq 'vhd') {
+                    Write-LogInfo "Generation 2 VM does not support vhd disk, please skip this case in the test script"
+                return $True
```

Comment:
$true

Reply:
Here $true, do you mean change $True to $true?  Seems that some scripts use $True, other scripts use $true, it does not keep consistent.
In LISAv2, keep using $true, correct?  Thank you so much.